### DISCUSSION TITLE
New function about playerObject

### DISCUSSION THREAD

**Discussion by guguxh** [wxyz-abcd/node-haxball#55] [General]
Adding properties to player object host-side, this is possible? Or this is just client-side?

**Comment by wxyz-abcd** [comment]
`onPlayerObjectCreated(playerObject)` callback works on both host-side and client-side, however I haven't tested it on client side. After thorough testing, if there are bugs, of course I will fix them.

**Reply by guguxh** [reply]
> `onPlayerObjectCreated(playerObject)` callback works on both host-side and client-side, however I haven't tested it on client side. After thorough testing, if there are bugs, of course I will fix them.

This will be useful to don't need other ways to store some data
